types/office-js/index.d.ts

```diff
@@ -131013,6 +131013,7 @@ declare namespace PowerPoint {
          *
          * @remarks
          * [Api set: PowerPointApi 1.8]
+         * This method is optimized to export a single slide. Exporting multiple slides can impact performance.
```

Comment:
I think you'll need to insert a blank comment line here so the page renders the previous link properly.

Also, do you need to add this remark to the office-js-preview version of this API?
 
```suggestion
         * 
         * This method is optimized to export a single slide. Exporting multiple slides can impact performance.
```